Server/Room.cpp

```diff
@@ -37,12 +38,24 @@
 #include "R-TypeLogic/Server/Systems/EnemiesGoRandom.hpp"
 #include "R-TypeLogic/Server/Systems/EnemyShootSystem.hpp"
 #include "R-TypeLogic/Server/Systems/LifeTimeDeathSystem.hpp"
+#include "R-TypeLogic/Server/Systems/RemoveAfkSystem.hpp"
 
 using namespace server_data;
 using namespace error_lib;
 using namespace communicator_lib;
 using namespace ecs;
 
+static server_data::Room::RoomState roomState(Room::RoomState::UNDEFINED);
+
+/// @brief Useful function called when a sigint received.
+/// @param signum Value of the received signal
+void signalCallbackHandler(int signum)
+{
+    (void)signum;
```

Comment:
Why not using it ?

Reply:
The signalCallBackHandler function is an overload of a current function (Need to respect a prototype).
The signum is not important for us, so we will simply ignore it :)